flake.nix

```diff
@@ -211,7 +189,7 @@
           in
             checksForNixosTests // checksForNixosExamples;
         in
-          concatMapAttrs checksForProject (importNgiProjects (pkgs // nonBrokenPackages));
```

Comment:
Note that this is not a problem. If we had broken packages filtered out that tests happen to rely on, everything would collapse anyway. We'd need to filter broken tests as well.